protoc-gen-openapiv2/internal/genopenapi/generator_test.go

```diff
@@ -0,0 +1,59 @@
+package genopenapi_test
+
+import (
+	"testing"
+
+	"github.com/grpc-ecosystem/grpc-gateway/v2/internal/descriptor"
+	"github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2/internal/genopenapi"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/descriptorpb"
+	"google.golang.org/protobuf/types/pluginpb"
+	"sigs.k8s.io/yaml"
```

Comment:
I just noticed, this looks like it's using the wrong yaml package

Reply:
well noticed, fixed